### ISSUE TITLE
Semver not respected

### ISSUE THREAD

**Issue by oliviertassinari** [zilverline/react-tap-event-plugin#38]
v0.1.9 introduce a breaking changes with v0.1.8.
http://semver.org/

**Comment by oliviertassinari** [comment]
can you unpublish v0.1.9 also. Thanks

**Comment by s0meone** [comment]
Yes I did.

```
-:react [master] $ npm unpublish react-tap-event-plugin@0.1.9
- react-tap-event-plugin@0.1.9
```

**Comment by s0meone** [comment]
I've also published 0.2.0, although npmjs.org doesn't show it yet.

```
-:react [master] $ npm publish .
+ react-tap-event-plugin@0.2.0
```

**Comment by oliviertassinari** [comment]
:+1:
